data/wanderer/wanderers start.txt

```diff
@@ -661,7 +625,7 @@ mission "Wanderers: Unfettered Diplomacy 1"
 		has "Wanderers: Defended Vara Ke'sok: done"
 	on offer
 		conversation
-			`On <origin>, you meet up with Iktat Rek and the Hai ambassador Sayari, and tell them that you just helped to drive off an Unfettered raid on the Wanderer water world of Vara Ke'sok. "The situation is not [sustainable, livable]," says Rek. "Perhaps the Unfettered are the ones [chosen, intended] to [inherit, receive] the fruit of our labors here, but until the Eye beckons us to a new [home, place] we have nowhere else to go."`
+			`On <origin>, you meet up with Iktat Rek and the Hai ambassador Sayari, informing them of your efforts in driving off an Unfettered raid on the Wanderer water world of Vara Ke'sok. Sayari herself now has a Wanderer translator too, although it seems to only translate to English, like yours. "The situation is not [sustainable, livable]," says Rek. "Perhaps the Unfettered are the ones [chosen, intended] to [inherit, receive] the fruit of our labors here, but until the Eye beckons us to a new [home, place] we have nowhere else to go."`
```

Comment:
```suggestion
			`On <origin>, you meet up with Iktat Rek and the Hai ambassador Sayari, informing them of your efforts in driving off an Unfettered raid on the Wanderer water world of Vara Ke'sok. Sayari herself now has a Wanderer translator too, although it seems to be a copy of yours as it's not translating to Hai. "The situation is not [sustainable, livable]," says Rek. "Perhaps the Unfettered are the ones [chosen, intended] to [inherit, receive] the fruit of our labors here, but until the Eye beckons us to a new [home, place] we have nowhere else to go."`
```

I don't like how it explicitly says English now, for the same reason I didn't like all the times the HR campaign mentioned "Human Standard".

Reply:
Agreed. I didn't notice that (and several spelling errors) in the corrections from TheMarksman-ES

Reply:
Wait. That's a thing now?
Then ya, use Human Standard.

Reply:
Mmm, it's a thing I like to do. I wouldn't use Human Standard though, I think it's very cliché to explain everything away with a universal language, and since at the time only HR had any reference to Human Standard, it didn't make sense that nobody in human space mentioned it at all. So to me it's better to just not say anything and imply it's saying something you understand.